packages/default/scss/menu/_layout.scss

```diff
@@ -202,12 +219,19 @@
 
 
     // RTL
-    .k-rtl {
+
+    .k-rtl,
+    [dir="rtl"] {
+
+        .k-menu:not(.k-menu-vertical) {
+            flex-direction: row-reverse;
```

Comment:
Why is this rule necessary? Flexbox containers automatically reverse the horizontal direction when in an RTL container ([see Dojo](http://dojo.telerik.com/ESUMitiT)). The horizontal RTL menu needs to be laid out in reverse order, and this rule prohibits that.